indico/modules/events/layout/models/menu.py

```diff
@@ -121,12 +153,21 @@ def default_data(self):
     def is_orphaned(self):
         return bool(self.name and self.default_data is None)
 
+    @property
+    def is_accessible(self):
```

Comment:
I think this should be a method that takes a `user` arg - it's a bit strange to use the session user for such things. and maybe we can use the more common `can_access` name?

Reply:
Should `is_visible` also take a `user` arg?

Reply:
Nevermind, I didn't notice that we already have existing code using it as well... I just didn't want to introduce it in new code, but I think for the sake of ocnsistency it's fine to leave it as is then.